src/plugins/Dashboard/index.js

```diff
@@ -333,7 +336,9 @@ module.exports = class DashboardUI extends Plugin {
     totalSize = prettyBytes(totalSize)
     totalUploadedSize = prettyBytes(totalUploadedSize)
 
-    const isAllComplete = state.totalProgress === 100
+    const isAllComplete = state.totalProgress === 100 &&
+      completeFiles.length === Object.keys(files).length &&
+      processingFiles.length === 0
```

Comment:
This needed some additional checks, because `totalProgress` refers to upload progress, but doesn't tell if postprocessing completed. I'm actually not sure if the `completeFiles.length === ` check is still necessary, it may be a leftover of my first shot at this, I can look into that tomorrow.